app/subscriber/src/features/router/AppRouter.tsx

```diff
@@ -35,6 +36,7 @@ export const AppRouter: React.FC<IAppRouter> = () => {
         <Route path="welcome" element={<AccessRequest />} />
         <Route path="access/request" element={<AccessRequest />} />
         <Route path="/landing/:id" element={<Landing />} />
+        <Route path="/search/:id" element={<SearchPage />} />
```

Comment:
`:id` parameter is a very misleading parameter.  Which caused me confusion when reviewing your above code.  This also should be a query string variable as mentioned above.